tests/Jellyfin.Dlna.Tests/ProfileTester.cs

```diff
@@ -0,0 +1,107 @@
+using Emby.Dlna;
+using Emby.Dlna.PlayTo;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller;
+using MediaBrowser.Model.Dlna;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Serialization;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Jellyfin.Dlna.Tests
+{
+    public class ProfileTester
```

Comment:
Alternatively if it becomes too unwieldy in the future, we can move the tests into a DlnaManager folder and rename the tests.
```suggestion
    public class DlnaManagerTests
```